truffle/src/main/ruby/core.rb

```diff
@@ -6,6 +6,14 @@
 # GNU General Public License version 2
 # GNU Lesser General Public License version 2.1
 
+module Truffle
+  # Utility method for commenting out part of Rubinius's implementation and detailing why.  It helps clearly
+  # demarcate things we want to omit from things Rubinius has left commented out and as such, should help compare
+  # diffs when upgrading Rubinius core files.
+  def self.omit(reason)
+  end
+end
```

Comment:
Yeah. The two cases I wanted to handle are:

1) Delineating stuff Rubinius has commented out from stuff we have commented out.
2) Annotating things we've essentially deleted (just deleting them means that needs to be reevaluated each time we sync upstream).

So, it's essentially documentation for why we can't run something while keeping the diff between different Rubinius files minimal.  It's also easy to grep for.